ci/rhel7/testing/saltstack-integration

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '600', daysToKeepStr: '7')),
+    disableConcurrentBuilds(),
+])
```

Comment:
Is there a way we can merge put all these common parts we use in all the pipelines files? Like include from an external file or something?

I see at the moment most of the code if duplicated across all the pipelines files. If this is easy and possible I would prefer to avoid duplicating code and only overriding default values in case a certain pipeline requires that.

Reply:
@meaksh yop goodpoint . there is for jenkins pipelines a way, which is not what i like most but we could try to use it.
I didn"t like it because in the past for jenkins-ci there was only global shared library.

more on topic:
https://jenkins.io/doc/book/pipeline/shared-libraries/

I don"t like the global shared library because you need to install them on jenkins.